src/static/js/modules/form-validation.js

```diff
@@ -6,34 +6,84 @@
 'use strict';
 
 var $ = require( 'jquery' );
+require( './jquery/cf_formValidator' ).init();
+require( './jquery/cf_notifier' ).init();
 
-function init() {
-  $( '.js-validate_form-not-empty' ).each( function() {
-    var $form = $( this );
+function _createMessage( type, label ) {
+  var message;
 
-    $form.on( 'submit', function() {
-      var formIsEmpty = true;
-      var event;
+  switch ( type ) {
```

Comment:
No, this function still has a complexity of 4. I'll create a bugfix for after the sprint is merged to map an object like you suggested. Not sure how I forgot about that in the cleanup.